tests/functional/adapter/test_model_grants_local.py

```diff
@@ -0,0 +1,170 @@
+import pytest
```

Comment:
This should be removed eventually.  Leaving in because when I run locally this fails in a different spot than the same test in the adapter zone.  Need more research.